yarn.lock

```diff
@@ -2306,7 +2306,7 @@ es6-error@^4.0.1:
   resolved "https://registry.yarnpkg.com/es6-error/-/es6-error-4.1.1.tgz#9e3af407459deed47e9a91f9b885a84eb05c561d"
   integrity sha512-Um/+FxMr9CISWh0bi5Zv0iOD+4cFh5qLeks1qhAopKVAJw3drgKbKySikp7wGhDL0HPeaja0P5ULZrxLkniUVg==
 
-es6-promise@^4.0.3:
+es6-promise@^4.0.3, es6-promise@^4.2.8:
```

Comment:
I wonder which library is keeping the `4.0.3` here?

Reply:
Looks like a deep dependency of codecov uses it too

Reply:
Output from `yarn why`:

```
=> Found "es6-promise@4.2.8"
info Has been hoisted to "es6-promise"
info Reasons this module exists
   - Specified in "dependencies"
   - Hoisted from "codecov#teeny-request#https-proxy-agent#agent-base#es6-promisify#es6-promise"
info Disk size without dependencies: "396KB"
info Disk size with unique dependencies: "396KB"
info Disk size with transitive dependencies: "396KB"
info Number of shared dependencies: 0
```